### ISSUE TITLE
Audio PT broken in 16.2.0

### ISSUE THREAD

**Issue by LePousson** [koying/SPMC#199]
I just discovered that the Audio PT does not work anymore since 16.2.0 ... Even when playing DTS HD, DOLBY TRUE HD, or simple DTS/DD 5.1, I get PCM stream ...

Tried both IEC & RAW PT but none worked ...

I had to reinstall 16.1.2 to get it working again ...

**Comment by war59312** [comment]
From ReadMe:

> Backport "disable PT when sync to display is enabled"

Just checking. Lots of us had Sync to display enabled it seems.

I thought the same thing until I realized this was the issue. Was seconds away from hitting submit myself but I thought I'd triple check.

**Comment by heymoe** [comment]
Yep, this is at least the 3rd "bug" report on here related to the Backport "disable PT when sync to display is enabled" in 16.2 that I've seen today on here. Please disable "Sync to Dispay" in Video Settings. If that still doesn't fix the issue for your then maybe you have a new bug but I think it will fix it.

**Comment by showmak** [comment]
I don't have problem with Audio PT.

**Comment by LePousson** [comment]
Ok The disable of "Sync to Display" fixed the PT thing ...

But now, I do not have any auto refresh rate working ...

EDIT : A full reboot of the SATV fixed that ... But I now have a small audio/video delay of 0.2s (200ms) every time I have a DD/DTS 5.1 or any HD stream

**Comment by ghost** [comment]
No log = no problem

**Comment by war59312** [comment]
Close? Been 3+ weeks with no log.

**Comment by LePousson** [comment]
= Been 3 weeks since I sent my SATV back to my reseller, not because of
SPMC but because of too many problems I got with it while everything is ok
with another player ...
